go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/windows-machine-config-operator
 
-go 1.20
+go 1.21
```

Comment:
1.20 should remain in 1.21

Reply:
reverting the go.mod changes caused build failures here as well. :thinking: 

error: GIMME_GO_VERSION not recognized as valid
  got: v1.20.12
Detected go version: go version go1.20.12 linux/amd64.
Kubernetes requires go1.22 or greater.
Please install go1.22 or later.